proto/process/agent.proto

```diff
@@ -230,6 +230,8 @@ message Process {
 	uint64 involuntaryCtxSwitches = 17;
 	bytes byteKey = 18;
 	bytes containerByteKey = 19;
+	int64 tcpConnections = 21;
```

Comment:
Couldn't we unsigned integers for these 2?
```suggestion
	uint64 tcpConnections = 21;
```

Reply:
You are right, I wasn't thinking too much but uint64 makes sense.